Makefile.am

```diff
@@ -92,8 +92,8 @@ TEST_EXTENSIONS = .sh
 
 .PHONY: update-version
 update-version:
-	./gen-version.sh >version.h.new
-	cmp version.h version.h.new && rm version.h.new || mv version.h.new version.h
+	$(top_srcdir)/gen-version.sh >version.h.new
+	cmp $(top_srcdir)/version.h version.h.new && rm version.h.new || mv version.h.new $(top_srcdir)/version.h
```

Comment:
Should version.h be in the top_srcdir? Looks like it's more appropriate to put it in the build dir?